src/styles/blocks/header.scss

```diff
@@ -0,0 +1,17 @@
+.header {
+  padding: 0 50px;
+  width: 100%;
+  display: flex;
+  background-color: #fff;
```

Comment:
lets try to organize you styles
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)